main.c

```diff
@@ -194,6 +222,14 @@ void get_options(int argc, char *argv[])
 		{
 			verbose = 1;
 		}
+		else if(strcmp(*argv, "-o") == 0)
+		{
+			overlay = 1;
+		}
+		else if(strncmp(*argv, "-m", 2) == 0)
+		{
+			delay = atol(*argv+2);
```

Comment:
You need to increment argv here to skip the delay argument afterwards

Reply:
I'd cheated and wasn't using a space between the option and value ("-m500" rather than "-m 500") so argv doesn't need to be incremented as it just skips over the 2 chars for "-m" before parsing the value.

I can update it to expect a space between the option and value.